pep-0001.txt

```diff
@@ -805,6 +808,10 @@ References and Footnotes
 
 .. _`GitHub issue`: https://github.com/python/peps/issues
 
+.. _Steering Council repository: https://github.com/python/steering-council
+
+.. _Steering Council issue: https://github.com/python/steering-council/issues/new/choose
```

Comment:
That repo doesn't have an issue chooser set up, I think (the link redirects to `https://github.com/python/steering-council/issues/new`)

```suggestion
.. _Steering Council issue: https://github.com/python/steering-council/issues/new
```

Reply:
Yes, but this is the canonical link for the <kbd>New Issue</kbd> button in GitHub's UI, and will automatically redirect to the chooser if one is set up (whereas cutting out `/choose` would ignore it even if it was set up, which is not likely what we want here), so the link should be kept how it is for forward compat and consistency with GitHub's UI, instead of trying to second-guess it.